app/models/planning_application.rb

```diff
@@ -1058,6 +1058,6 @@ def filter_documents_for_tab(documents, tab)
   end
 
   def neighbour_review_requested?
-    consultation.present? && consultation.reviews.any? { |review| review.to_be_reviewed? }
+    consultation.present? && consultation.reviews.any?(&:to_be_reviewed?)
```

Comment:
Returning `nil` should be fine and treated as falsey though it can be contentious - there was a Rails PR that ran to 96 comments about making `request.xhr?` return true/false before being closed unmerged 😂